src/test/java/ebl/v2/ShipmentEventIT.java

```diff
@@ -116,8 +126,8 @@ void testGetAllEventsByCarrierBookingReference() {
       .body("eventType", everyItem(equalTo("SHIPMENT")))
       .body("eventClassifierCode", everyItem(equalTo("ACT")))
       .body("documentTypeCode", everyItem(anyOf(equalTo("SHI"), equalTo("TRD"))))
-    // TODO: Assert that document references contain the carrier booking reference 832deb4bd4ea4b728430b857c59bd057
-    // (Needs a fix for DDT-928 first)
+      .body("documentReferences.flatten().findAll { it.documentReferenceType == 'BKG' }.size()", greaterThanOrEqualTo(3))
+      .body("documentReferences.flatten().findAll { it.documentReferenceType == 'BKG' }.documentReferenceValue", everyItem(equalTo("832deb4bd4ea4b728430b857c59bd057")))
```

Comment:
missing json schema validation

Reply:
The alleventsand headers test already have
.body(jsonSchemaValidator("shipmentEvent"));

Should we put schemavalidations in every test?